src/webprojects/relay.md

```diff
@@ -0,0 +1,80 @@
+
+* In Pontoon: Ping the l10n PM in charge in the comment field on the right side of the translation field.
+* Matrix channel: Ping the l10n PM in charge with any issues you may have.
+* Localization on Discourse: Follow announcements and discussions, and raise questions there.
+
+### To Test
+
+Firefox Relay Website is synced once a day between localized and the core repo. To test your localization, you need to download the build at https://github.com/mozilla/fx-private-relay/releases. Click on the Assets arrow, you should see two files. Download the appropriate compressed file to your machine. To download the latest l10n build, click on the Tags button (next to the Release button), the latest build is on the top.
+
+There are three level of testings:
+
+* The free service can be tested by anyone. All you need is a Firefox account. You can have up to 5 free email masks.
+* The premium service is a paid service. It allows unlimited email masks. Service is available to select markets. Market expansions continue.
+* The phone masking service is also a paid service. It allows one phone mask per phone number. Service is limited to select markets.
+
+**NOTE** for paid service testing:
+* The product team is currently working on a testing solution for markets where premium services are offered. Stay tuned for the update. At the moment, even though the strings related to paid services are translated, only those on production site in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) can be tested.
+
+#### Linguistic testing
+
+* Login screen.
+* Email notification.
+* Phone message notification (if feature available).
+* Check all the setting configuration buttons and the page or expanded window each page leads to.
+* Translation is of good quality and makes sense in context.
+* No broken UI, especially in a mobile environment. Text wraps or line breaks correctly.
+* Terms are consistent between pages.
+* Basic font support.
+* Error and warning messages.
+
+#### Functionality testing
+
+* Configure the Settings. Go through all the applicable categories.
+* Check for email notifications with each of the initial setups.
+* Make changes to some of the settings, check for possible error messages or warnings.
+* Check for followup confirmation emails and phone messages (if applicable).
+
+## Firefox Relay Add-on
+
+Production: https://addons.mozilla.org/firefox/addon/private-relay/
+Dev build: https://github.com/mozilla/fx-private-relay-add-on/releases
+
+This is a companion project to Firefox Relay Website project. The sync between localized and the core repo is once a day.
+
+### To Test
+
+You need to download the build at https://github.com/mozilla/fx-private-relay-add-on/releases. Click on the Assets arrow, and you should see three files. Install the latest build, then download the appropriate compressed file to your machine. The build is manual and is done every few days for the time being.
```

Comment:
I haven't gone through the install/build process for this one, but I'm going to assume my comments for `fx-private-relay` also applies here.